charts/incubator/channels-dvr/values.yaml

```diff
@@ -3,6 +3,11 @@ image:
   pullPolicy: Always
   tag: latest@sha256:bfc3c73c1cefd578c121b6e3256a496ad9ee190a6a6901d3ad1b9aa2a7be082c
 
+tveImage:
+  repository: fancybits/channels-dvr
+  pullPolicy: Always
+  tag: tve@sha256:0fb54da902ae9bb872634e38a653f66c2698373ab25c587d79d41f24271077cb
```

Comment:
```suggestion
  repository: tccr.io/truecharts/channels-dvr-tve
  pullPolicy: Always
  tag: latest@sha256:6edc4e63493996747bf07f5f58d4ec0e8bfc2ed22fe2c5c27706a7c720e6a8a1
```

Reply:
Is the TrueCharts mirror converting the `tve` tag to `latest`?

Reply:
Yes because it's under another "repo" `/chanels-dvr-tve`

Reply:
Where does the new hash come from?